src/App.js

```diff
@@ -2,19 +2,27 @@ import styled from 'styled-components';
 import {Routes, Route} from 'react-router-dom';
 import FormPage from './pages/FormPage';
 import CalendarPage from './pages/CalendarPage';
+import NavTop from './components/NavTop';
+import NavBottom from './components/NavBottom';
+import {loadFromLocalStorage, writeToLocalStorage} from './util/localstorage';
```

Comment:
GitHub Actions tells me these two functions are never used - do you really need them here?